### ISSUE TITLE
Error after installing via Package Control

### ISSUE THREAD

**Issue by rudedogg** [ziglang/sublime-zig-language#46]
I keep getting this error popup after installing the plugin via package control:

```
error loading meta info from Packages/Zig Language/Preferences/Comments.tmPreferences: Error parsing plist xml: expected < in file Packages/Zig Language/Preferences/Comments.tmPreferences on line: 1
```

The error seems to happen when I interact with Package Control (add/remove packages). Syntax highlighting still works, so it's not a big deal.

I'm using the master branch of Package Control, and the beta version of macOS 11 – so this could definitely be an issue on my end and I just need to wait. Is anyone else having this happen?

**Comment by lzutao** [comment]
I have the same error dialog in Linux amd64.
![image](https://user-images.githubusercontent.com/15225902/97178373-f5860b00-17c9-11eb-8092-7859112297da.png)

**Comment by dsego** [comment]
It looks like this is not happening anymore for me on Sublime build 4092.

**Comment by haferburg** [comment]
Same here on Windows.

**Comment by brodeuralexis** [comment]
Did some investigating, `Preferences/Comments.tmPreferences` looks like this from a fresh install.
Was there an error when packaging ?
![zig-comments-preference](https://user-images.githubusercontent.com/11179144/103163574-613a2380-47cd-11eb-80aa-20fd3b22070a.png)

**Comment by naringas** [comment]
I just got this error on Sublime Text after a fresh install of this via package control.

I can confirm that the workaround (from https://github.com/ziglang/sublime-zig-language/issues/46#issuecomment-751423064 ) does seems to work. I was able to do it by modifying the zipped up  `.config/sublime-text-3/Installed Packages/Zig Language.sublime-package`

**Comment by Cobertos** [comment]
Workaround worked for me as well. Unzip, change the file, rezip as `.zip`

**Comment by Manuzor** [comment]
I do not encounter the problem when cloning the repo into `~\AppData\Roaming\Sublime Text 3\Packages\` on windows. So yea, seems like the version uploaded to package control is outdated.

**Comment by ron-wolf** [comment]
For anyone wondering: Sublime packages are zip archives, as you’ll find by running `file "$HOME/Application Support/Sublime Text 3/Installed Packages/Zig Language.sublime-package"`. As such, fixing the issue can involve a bit of tedious decompressing and recompressing. The following script should suffice as a temporary fix for an individual installation (it prints stuff verbosely so you know what’s going on, and exits on the first error):

```bash
#!/usr/bin/env bash -e

ZIP_DEST="$(mktemp -d)"
ZIP_SRC="$HOME/Library/Application Support/Sublime Text 3/Installed Packages/Zig Language.sublime-package"

unzip "$ZIP_SRC" {Preferences,Support}/Comments.tmPreferences -d "$ZIP_DEST"
tee <"$ZIP_DEST/Support/Comments.tmPreferences" "$ZIP_DEST/Preferences/Comments.tmPreferences"

pushd "$ZIP_DEST"
zip -Tv "$ZIP_SRC" Support/Comments.tmPreferences
popd

rm -Rf "$ZIP_DEST"
```

**Comment by kevinsjoberg** [comment]
I've released a new version, [1.5.0](https://github.com/ziglang/sublime-zig-language/releases/tag/1.5.0), that is [semver](https://semver.org) compatible. It's now available in Package Control. It resolves this issue.

Enjoy!
